src/archive/2022-2023/demos/README.md

```diff
@@ -0,0 +1,15 @@
+# Demos
+
+At Tech Start UCalgary
+all projects are showcased to the world
+at the end of each academic year.
+In this event, people from different industries and backgrounds
+come and give us feedback on the projects,
+while also allowing you as a professional and us as a club
+to connect and network with different companies.
+
+This year's showcase is scheduled for **April 29th, 2023**.

```

Comment:
April 27 this year!

Reply:
These are in the archive, so I kept them the same as last year 👁️